pkgs/by-name/re/resumed/package.nix

```diff
@@ -2,20 +2,46 @@
   lib,
   buildNpmPackage,
   fetchFromGitHub,
+  nix-update-script,
+  testers,
+  chromium,
 }:
 
-buildNpmPackage rec {
+buildNpmPackage (finalAttrs: {
   pname = "resumed";
-  version = "4.1.0";
+  version = "6.0.0";
 
   src = fetchFromGitHub {
     owner = "rbardini";
     repo = "resumed";
-    rev = "v${version}";
-    hash = "sha256-kDv6kOVY8IfztmLeby2NgB5q0DtP1ajMselvr1EDQJ8=";
+    rev = "v${finalAttrs.version}";
+    hash = "sha256-K9F6ZxtqAQSc5Dqeoysish+xeRqDcDG/6Ynx7bTJfl8=";
   };
 
-  npmDepsHash = "sha256-7Wdf8NaizgIExeX+Kc8wn5f20al0bnxRpFoPy6p40jw=";
+  npmDepsHash = "sha256-UElS1pEzPv0FnvMGCnqEFBi7JzE8QWRFynkAPHy35FY=";
+
+  postPatch = ''
+    sed -i 's/"version": ".*"/"version": "${finalAttrs.version}"/' package.json
+  '';
+
+  dontNpmPrune = true;
```

Comment:
This seems like a very expensive way to make the puppeteer package available, but it works. I did not find a better way (yet), but I'm also not familiar with npm packaging in nixpkgs.

Reply:
Hi, thanks a lot for looking into this!

Overall there were 3 issues I fixed:
* a build issue caused by puppeteer fetching chromium during build, where `PUPPETEER_SKIP_DOWNLOAD`was the fix
* the puppeteer npm pckage was not found during `resumed export`, because it was installed but later removed again in the `npm prune` step, which is fixed by the dontNpmPrune
* chromium was not found during runtime, which was fixed by the `wrapProgram` call

So apart from a typo I just fixed and force-pushed, all seems to be fine and work correctly.